users/src/controllers/user.controller.ts

```diff
@@ -153,3 +165,44 @@ export const findRelation = async (req: Request, res: Response) => {
     }
     res.status(200).send(response);
 }
+
+/**
+ * Method for editing any field of the user with the specified id.
+ * Expects req.body to have a user object
+ * If all checks pass, will update User with the fields specified
+ * Then sends 200 alongside the newly updated user object.
+ * @param req 
+ * @param res 
+ */
+export const editUser = async (req: Request, res: Response) => {
+    if(!req.params.userId) throw new BadRequestError("Please enter a user ID!");
+    const user = await User.findOne({ where: { id: req.params.userId } });
+    if(!user) throw new NotFoundError();
+    let imagePath = "";
+    if(req.file){
+        imagePath = req.file.path;
+    }
+ 
+    if(isEmpty(req.body)) throw new BadRequestError("Empty request!");
```

Comment:
not necessary if validator is used

Reply:
Which would be true except I put optional() tag on all validators meaning that if the field is entirely empty it will skip all validation and as such not throw an error

Reply:
and nothing is updated, it just used the same data. So, no need to check anyway

Reply:
ok then